Tests/SwiftParserTest/AttributeTests.swift

```diff
@@ -533,10 +533,10 @@ final class AttributeTests: XCTestCase {
       """,
       diagnostics: [
         DiagnosticSpec(message: "expected ':' and string literal in @_unavailableFromAsync argument", fixIts: ["insert ':' and string literal"]),
-        DiagnosticSpec(message: #"unexpected code '= "abc"' in attribute"#),
+        DiagnosticSpec(message: #"unexpected code '= "abc"' in attribute"#, fixIts: [#"remove '= "abc"'"#]),
       ],
       fixedSource: """
-        @_unavailableFromAsync(message: ""= "abc")
+        @_unavailableFromAsync(message: "")
         func foo() {}
         """
     )
```

Comment:
My work have affected the `@_unavailableFromAsync` test case.
It seems that a diagnosis should be generated that replaces `=` with `:` instead of removing `abc` 🤔 
I have only modified the result to ensure that it does not fail, as this issue is being tracked in #1606.

Reply:
This is properly another bug in the parser.
I think it's not adding a missing `:` when it comes to the `= abc`

Reply:
A-ha! Thank you for explaining.